actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -431,6 +431,10 @@ def ssl?
       super || scheme == "wss"
     end
 
+    def inspect # :nodoc:
+      "<##{self.class.name} method=#{method}, original_url=#{original_url}, remote_ip=#{remote_ip}, media_type=#{media_type}>"
```

Comment:
Can you change these to add quotes around them?  I think like this:
```suggestion
      "<##{self.class.name} method=#{method.dump}, original_url=#{original_url.dump}, remote_ip=#{remote_ip.dump}, media_type=#{media_type.dump}>"
```

I haven't tested the patch so I'm not 100% sure it's right.

Reply:
I've made the change.